.github/workflows/cleanup_caches.yml

```diff
@@ -1,14 +1,16 @@
 name: Cleanup caches
 on:
   workflow_dispatch:
+  pull_request:
```

Comment:
Should it be run on every PR? or is it for testing only?

Reply:
yes, that is for testing only